types/sceditor/index.d.ts

```diff
@@ -0,0 +1,106 @@
+// Type definitions for SCEditor 2.1.2
+// Project: https://www.sceditor.com/
+// Definitions by: Martin Pfund
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+// Work In Progress, may contains errors
+
+// declare module sceditor {
+declare module app.common.sceditor {
+    interface ISCEditor {
+        dom: IDOM;
+        command: ICommand;
+
+        create(selector: HTMLElement, args?: ISCEditorOptions): void;
+        instance(selector: HTMLElement): ISCEditorInstance;
+    }
+
+    interface ICommand {
+        set(command: string, options: ICommandOptions): void;
+    }
+
+    interface ICommandOptions {
+        exec?: (caller: HTMLElement) => void;
+        tooltip?: string;
+    }
+
+    interface ISCEditorOptions {
+        plugins?: string;
+        format?: string;
+        toolbar?: string;
+        locale?: string;
+    }
+
+    interface ISCEditorInstance {
+        // _(...args: string[]): string;
```

Comment:
Are are all of these commented out?

Reply:
Yes, this is a first start only. I plan to put more definitions into it.